test/support/helpers/return-log.helper.js

```diff
@@ -4,11 +4,11 @@
  * @module ReturnLogHelper
  */
 
+const ReturnLogModel = require('../../../app/models/return-log.model.js')
```

Comment:
Not sure why the position of the `requires` changed. They looked to be in the right positions prior to this change.